R/board_s3.R

```diff
@@ -221,6 +221,34 @@ pin_store.pins_board_s3 <- function(board, name, paths, metadata,
   name
 }
 
+#' @rdname board_deparse
+#' @export
+board_deparse.pins_board_s3 <- function(board, ...) {
+  bucket <- check_board_deparse(board, "bucket")
+  if ("prefix" %in% names(board)) {
+    prefix <- board[["prefix"]]
+  } else {
+    prefix <- NULL
+  }
+
+  ## non-auth config
+  config <- board$svc$.internal$config
+  region <- config$region
+  if (nchar(region) == 0) region <- NULL
```

Comment:
```suggestion
  region <- if (nchar(region) == 0) NULL else region
```

I think this form is fine for simple expressions